ansible/group_vars/nimbus.mainnet.yml

```diff
@@ -121,7 +121,15 @@ beacon_node_metrics_port:   '{{ 9200 + idx|int + 1 }}'
 beacon_node_rest_address:   '0.0.0.0'
 beacon_node_max_peers: '{{ node.get("max_peers", 320) }}'
 # Firewall
-beacon_node_firewall_libp2p_open: '{{ node.get("open_libp2p_ports", true) }}'
+# test mainnet nimbus with closed libp2p port
+beacon_node_closed_libp2p_ports: >-
+  {%- set closed_ports = [] -%}
+  {%- for config in nodes_layout[inventory_hostname] if config.get('close_libp2p_port', false) -%}
+      {%- set port = 9000 + loop.index0 + 1 -%}
+      {%- set _ = closed_ports.append(port|string) -%}
+    {%- endif -%}
+  {%- endfor -%}
+  {{ closed_ports|join(',') }}
```

Comment:
Not bad. Could have done the same with a lookup or filter plugin, but this is good.